src/pages/_app.tsx

```diff
@@ -54,8 +55,12 @@ Router.events.on('routeChangeStart', () => {
 Router.events.on('routeChangeComplete', () => NProgress.done())
 Router.events.on('routeChangeError', () => NProgress.done())
 
-// eslint-disable-next-line import/no-default-export
-export default function App({ Component, pageProps }: AppProps) {
+interface CustomAppProps {
+  Component: NextComponentType<NextPageContext, any, InitialProps>
+  pageProps: InitialProps
+}
```

Comment:
Using normal AppProps should suffice here. (InitialProps will be removed soon and replaced by page specific prop types)